draft-js-emoji-plugin/src/components/EmojiSelect/index.js

```diff
@@ -45,17 +47,18 @@ export default class EmojiSelect extends Component {
   // When the selector is open and users click anywhere on the page,
   // the selector should close
   componentDidMount() {
-    document.addEventListener('click', this.closePopover);
+    document.addEventListener('click', this.closeIfClickedOutside);
   }
 
   componentWillUnmount() {
-    document.removeEventListener('click', this.closePopover);
+    document.removeEventListener('click', this.closeIfClickedOutside);
   }
 
-  onClick = e => {
-    e.stopPropagation();
-    e.nativeEvent.stopImmediatePropagation();
```

Comment:
I'd prefer it if those are kept, to make sure the behaviour doesn't change

Reply:
thanks for reviewing it @juliankrispel, the unique purpose of the portion of the code you have mentioned is to prevent EmojiSelect from closing when clicking on something thats inside it, but with this modifications its no longer needed cause now it will be handled by this portion.

```
  componentDidMount() {
    document.addEventListener('click', this.closeIfClickedOutside);
  }

&

  closeIfClickedOutside = e => {
    if (this.emojiSelectRef && !this.emojiSelectRef.current.contains(e.target)) {
      this.closePopover();
    }
  }
```

What it actually does it checks if the random click target is outside of the EmojySelect component and closes it. 

If I leave that it will prevent other open instances of EmojySelect from being closed when you open another one.
So imagine I open an instance of EmojySelect, it opens up, then I open another one without closing the first one, the stop propogation will prevent the click bubbling so it want reach the document and already opened instance of EmojySelect will simply not close.

But still as I have already highlighted that portion is simply not needed because of use of `closeIfClickedOutside` function.

let me know what you think ;)